src/main/java/com/ericsson/ei/queryservice/ProcessQueryParams.java

```diff
@@ -76,13 +80,44 @@ public JSONArray filterFormParam(JSONObject criteriaObj, JSONObject optionsObj)
             String result = "{ \"$and\" : [ " + criteria + "," + options + " ] }";
```

Comment:
should "result" variable name something else, since its not a result of processQueryAgg...() function.

Reply:
Fixed, changed to request